Tests/Unit/Pipeline/PipelineTest.php

```diff
@@ -82,13 +82,11 @@ public function testPipeline($data, $expectedConsumedItems, $expectedSkippedItem
      * OnModify.
      *
      * @param ItemPipelineEvent $event
-     *
-     * @throws ItemSkipException
      */
     public function onModify(ItemPipelineEvent $event)
     {
         if ($event->getItem() == 'skip') {
-            throw new ItemSkipException();
+            ItemSkipper::skip($event, 'reason');
```

Comment:
in production it would be something like `"Item #${event->getItem()->getId()} skipped: not in stock"`, but in unit test such verbosity is a bit dubious.